tests/components/deconz/test_light.py

```diff
@@ -117,6 +126,15 @@ async def test_lights_and_groups(hass):
     assert tunable_white_light.attributes["min_mireds"] == 155
     assert tunable_white_light.attributes["supported_features"] == 2
 
+    tunable_white_light_bad_maxmin = hass.states.get(
+        "light.tunable_white_light_with_bad_maxmin_values"
+    )
+    assert tunable_white_light_bad_maxmin.state == "on"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)